src/store/stub/store.py

```diff
@@ -24,16 +31,16 @@ def setup(self):
             self.publishers = json.load(f)
 
         with open(Path(content_dir / "topics.json").absolute()) as f:
-            self.topics = json.load(f)
+            self.themes = json.load(f)
 
-    def get_dataset(self, id: str) -> List[dict]:
```

Comment:
go with get_dataset()l get_topic() and get_publisher() please (no mention of id in the method name anymore). This is another one I changed on a different pr.